src/components/safe-apps/SafeAppsHeader/styles.module.css

```diff
@@ -2,3 +2,7 @@
   background: var(--color-background-paper);
   padding: var(--space-2) var(--space-3);
 }
+
+.searchInput {
+  width: 716px;
```

Comment:
Also a good number, why exactly 716px and how will it look on mobile?

Reply:
It's not a min-width, so it will shrink on mobile. The number comes from design, I can put it into a variable, but I couldn't foresee a need to use the number outside, so decided to hardcode it

![image](https://user-images.githubusercontent.com/16622558/186171452-3f8bc2de-5e09-46ac-a7e7-3a35be5f29e6.png)

Reply:
I don't think the design meant exactly 716px. 80% or something like that would be good enough.

Reply:
Fair enough 👍

Reply:
Turns out this style isn't used anymore, one weak side of separate CSS files is dead code elimination :(